internal/super/flix.go

```diff
@@ -161,3 +185,25 @@ func execute(
 	}
 	return transactions.SendTransaction([]byte(cadenceWithImportsReplaced), args[1:], "", flow, state, transactionFlags)
 }
+
+func (fr *flixResult) JSON() any {
+	result := make(map[string]any)
+	result["output"] = fr.output
+	result["save"] = fr.save
+	return result
+}
+
+func (fr *flixResult) String() string {
+	return isSaved(fr)
+}
+
+func (fr *flixResult) Oneliner() string {
+	return isSaved(fr)
+}
+
+func isSaved(fr *flixResult) string {
+	if fr.save != "" {
+		return fmt.Sprintf("Generated code saved to %s", fr.save)
```

Comment:
Use the logger instead of fmt

Reply:
the isSaved method returns a string. It's a convenience method for `String()` and `Oneliner()` methods. it's not a log out statement specifically.